charts/nebula-crawler/templates/cron.yaml

```diff
@@ -0,0 +1,220 @@
+# CRAWL
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-crawl
+spec:
+  schedule: {{ .Values.crawler.schedule }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: nebula-crawler
+            chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+            heritage: {{ .Release.Service }}
+            release: {{ .Release.Name }}
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - name: crawl
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+              ports:
+                - containerPort: 6666
+                  name: metrics
+              command: [ "nebula" ]
+              args: [ "crawl" ]
+              env:
+                {{- if .Values.postgres.enabled }}
+                - name: NEBULA_DATABASE_HOST
+                  value: {{ .Values.postgres.teamid }}-{{ .Release.Name }}-crawldb
+                - name: NEBULA_DATABASE_PORT
+                  value: "5432"
+                - name: NEBULA_DATABASE_NAME
+                  value: {{ .Values.postgres.database }}
+                - name: NEBULA_DATABASE_USER
+                  value: {{ .Values.postgres.user }}
+                - name: NEBULA_DATABASE_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Values.postgres.user }}.{{.Values.postgres.teamid }}-{{ .Release.Name }}-crawldb.credentials.postgresql.acid.zalan.do
+                      key: password
+                - name: NEBULA_DATABASE_SSL
```

Comment:
If you changed the env name on the other repo this var needed to be `NEBULA_DATABASE_SSL`